ssl/statem/extensions.c

```diff
@@ -54,13 +52,14 @@ typedef struct extensions_definition_st {
      */
     int (*init)(SSL *s, unsigned int context);
     /* Parse extension sent from client to server */
-    int (*parse_ctos)(SSL *s, PACKET *pkt, int *al);
+    int (*parse_ctos)(SSL *s, PACKET *pkt, X509 *x, size_t chain, int *al);
```

Comment:
I still disagree.  size_t is for the *size* of something, not an index.  int is appropriate.  And chain is still a bad name, idx or index or which is better.